src/runtime_src/core/pcie/tools/xbutil/xbutil.cpp

```diff
@@ -2531,6 +2394,8 @@ int xcldev::xclScheduler(int argc, char *argv[])
         {"echo", required_argument, 0, 'e'},
         {"kds_schedule", required_argument, 0, 'k'},
         {"cu_intr", required_argument, 0, xcldev::KDS_CU_INTERRUPT},
+        {"test", required_argument, 0, xcldev::KDS_TEST},
```

Comment:
Is scheduler a hidden option? xbutil help doesn't show it

Reply:
Yes. "xbutil scheduler" is hidden. Only use for internal debugging.